scripts/bake-ide-ami.ps1

```diff
@@ -898,6 +899,7 @@ $jsonObject = @"
             Sleep -Seconds 10
         }
         Write-Host "$(Log-Date) AMI $amiID is available"
+        Write-Host "##vso[task.setvariable variable=amiID;isOutput=true]'$amiID'"
```

Comment:
Please explain this code to me.

Reply:
@robe070 In Azure pipeline just like as we use ImageURI for VM tests, Varun told that it is amiID in case of AWS.  I made that change so that amiId will be avialable in the pipeline.

Reply:
I've not seen the syntax before. I think it should be conditioned on $Pipeline -eq $True

Reply:
I just copied the line from azure part. It's working fine in the pipeline.

Reply:
I can see it's value in further tasks in the pipeline.

Reply:
And when its not running in the pipeline...

thats the point

Reply:
I just checked it. If it is not running in pipeline, it will just print that line.